notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-761_vms_io_and_plotting.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<p>What does the x axis show? sorry I am bit rusty on my spectrograms</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
ah the x axis is seconds, I have added a label to clarify